design/BulkAPI.md

```diff
@@ -216,28 +237,28 @@ resource optimization in Kubernetes environments. Below is a breakdown of the JS
 
     - Each object in the `experiments` array has the following structure:
 
-  | Field                   | Type         | Description                                                              |
-              |-------------------------|--------------|--------------------------------------------------------------------------|
-  | `name`                  | `string`     | Name of the experiment, typically indicating a service name and deployment context. |
-  | `notification`          | `object`     | Notifications specific to this experiment (if any).                      |
-  | `recommendation`        | `object`     | Recommendation status and notifications specific to this experiment.     |
+  | Field            | Type     | Description                                                                         |
+  |------------------|----------|-------------------------------------------------------------------------------------|
+  | `name`           | `string` | Name of the experiment, typically indicating a service name and deployment context. |
+  | `notification`   | `object` | Notifications specific to this experiment (if any).                                 |
+  | `recommendation` | `object` | Recommendation status and notifications specific to this experiment.                |
```

Comment:
Can you please change this to `notifications` and `recommendations` everywhere, thanks

Reply:
Done